Introduction
============

The genome assemblies used by the [Genomics Education
Partnership](https://thegep.org) (GEP) scientific projects have not undergone
manual sequence improvement, and they might contain consensus errors (i.e.
substitutions, insertions, and deletions) that impact the annotation of
protein-coding genes. For example, the [Baylor College of Medicine Human Genome
Sequencing Center](https://www.hgsc.bcm.edu/) (BCM-HGSC) has previously
sequenced eight *Drosophila* species using the 454 and Illumina sequencing
platforms. The 454 sequencing technology exhibits low accuracy in long (\> 6bp)
mononucleotide runs. Similarly, sequencing reads produced by the Pacific
Biosciences (PacBio) and Nanopore sequencing platforms have lower read accuracy
(\~90%) and higher rates of insertions and deletions compared to Sanger
sequencing data.

The GEP developed the [*Sequence
Updater*](https://gander.wustl.edu/%7ewilson/sequence_updater/index.html) tool
to provide a standardized way to document consensus errors in the project
sequence using the [Variant Call
Format](https://samtools.github.io/hts-specs/VCFv4.2.pdf) (VCF). The VCF files
generated by the *Sequence Updater* can be used in conjunction with the [*Gene
Model Checker*](https://gander.wustl.edu/~wilson/genechecker/index.html) to
verify gene models that contain consensus sequence errors. Multiple VCF files
can be combined into a single project VCF file using the [*Annotation Files
Merger*](https://gander.wustl.edu/~wilson/submissionhelper/index.php) in
preparation for project submission.

Please see the [*Sequence Updater* User
Guide](https://community.gep.wustl.edu/repository/documentations/Sequence_Updater_User_Guide.pdf)
for an overview of the program, and some examples on how to use this program in
practice.



Availability
============

The [*Sequence
Updater*](https://gander.wustl.edu/%7ewilson/sequence_updater/index.html) is
available under the "**Resources & Tools**" section of the [F Element project
page](https://thegep.org/felement/) and the [Pathways project
page](https://thegep.org/pathways/) on the GEP website.



External Dependencies
=====================

* A local [mirror of the *UCSC Genome
  Browser*](https://genome.ucsc.edu/goldenPath/help/mirror.html)
* [*Samtools*](https://www.htslib.org/)
* [*VCFtools*](https://vcftools.github.io/index.html)
